requirements.txt

```diff
@@ -9,7 +9,7 @@ pathlib2==2.3.2 --hash=sha256:8e276e2bf50a9a06c36e20f03b050e59b63dfe0678e3716433
 python-dateutil==2.7.5 --hash=sha256:56f285e7fad54cde3e31dc68a31a861543bfee5ada9278da8e85ec20a8f72912
 python-editor==1.0.3 --hash=sha256:44fc57a6db6e04c7922c37a04d0a86d0024a4f0f06245b6c57638cb322176202
 requests==2.20.0 --hash=sha256:2a539dd6af40a611f3b8eb3f99d3567781352ece1698b2fab42bf4c2218705b5
-securedrop-sdk==0.0.4 --hash=sha256:50a6b980e2fcdbc78b00fa7b978cc91350364c94497f730b1c01f5ad446866c7
+securedrop-sdk==0.0.8 --hash=sha256:eca697da2520b568a4600b11c3ab818f5855e0ce13f88495d756a894f142f2a1
```

Comment:
This will be updated during the release process, and will contain the hash of the wheel on FPF's pip mirror, see explanatory gif here: 

https://raw.githubusercontent.com/freedomofpress/securedrop-debian-packaging/b4c640cb69b48f190b07fa803a5100bc4d8e8511/images/securedrop-pip-mirror.gif

Reply:
Right. Removed.